src/Accordion/stories/Accordion.stories.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import React from 'react'
+
+import { Accordion } from '../Accordion'
+import { AccordionItem } from '../AccordionItem'
+
+export default {
+  title: 'components/Accordion'
+}
+
+export const Default = () => {
+  const props = {}
+
+  return (
+    <Accordion {...props}>
+      <AccordionItem header='Default'>Item 1</AccordionItem>
+      <AccordionItem header='Opened' isOpen>
+        Item 2
+      </AccordionItem>
+      <AccordionItem header='Disabled' isDisabled>
+        Item 3
+      </AccordionItem>
+    </Accordion>
```

Comment:
Should we also add a story for the case where an accordion is both open and disabled?

Reply:
Does it make sense for things to be open and disabled?

Reply:
I can't think of a real world situation where that would apply. 
However, the component does allow for it, so it would make sense to represent it in the story.
But that's just me being picky.